src/instantiator/characteristic/enumeration-characteristic-instantiator.ts

```diff
@@ -113,7 +113,13 @@ export class EnumerationCharacteristicInstantiator extends CharacteristicInstant
             // create the related instance and attach the metamodel element to it
             const entityInstance = new DefaultEntityInstance(quad.object.value.split('#')[1], entity, descriptions);
             entityInstanceQuads.forEach(quad => {
-                entityInstance[quad.predicate.value.split('#')[1]] = quad.object.value;
+                if (Util.isBlankNode(quad.object)) {
```

Comment:
For more readability add these here:

`                const predicateKey = this.getPredicateKey(quad);

                if (Util.isBlankNode(quad.object)) {
                    const resolvedBlankNodes = this.metaModelElementInstantiator.rdfModel.resolveBlankNodes(quad.object.value);
                    entityInstance[predicateKey] = this.solveBlankNodeValues([...resolvedBlankNodes]);
                } else {
                    entityInstance[predicateKey] = this.createLanguageObject(quad);
                }`